schema-engine/sql-migration-tests/tests/initialization/mod.rs

```diff
@@ -53,10 +53,10 @@ fn connecting_to_a_postgres_database_with_missing_schema_creates_it(api: TestApi
                 "#
         );
 
-        let me = migration_api(Some(schema.clone()), None).unwrap();
+        let me = schema_api(Some(schema.clone()), None).unwrap();
```

Comment:
Is `me` supposed to stand for "migration engine"? This reads a little unclearly to me

Reply:
We seem to use this in other areas too

Reply:
Should maybe rename to `engine`... It's test code so I'm not too eager to do that work.

Reply:
cool, no worries